pkg/runtime/javascript/javascript.go

```diff
@@ -32,9 +32,16 @@ func init() {
 // Code template.
 var code = template.Must(template.New("js").Parse(`{{.Comment}}
 
+// Put the main logic of the task in this function.
 export default async function(params) {
   console.log('parameters:', params);
-  return params;
+
+  // You can return data to show outputs to users.
+  // Documetation on outputs: https://docs.airplane.dev/tasks/outputs.
```

Comment:
```suggestion
  // Outputs documentation: https://docs.airplane.dev/tasks/outputs
```

(but also typo in documetation)